courses/static/header.md

```diff
@@ -0,0 +1,20 @@
+<h3 align="center">SKIP – Past Exams Repository</h3>
+<p align="center">
+  <img align="center" src="docs/assets/skip-past-exams-poster.png"/>
+</p><br>
+
+Welcome to the official `GitHub` repository that gathers **examination materials** for programmes at the **IT Faculty** of [Gothenburg University](https://www.gu.se/).
```

Comment:
Right. We can leave it with *IT Faculty* for now, however, we'll need to change this later.

Reply:
I reckon this may be outside this PR, just noting it down, so we don't forget later.